### DISCUSSION TITLE
Petition: Request to use actual daylight

### DISCUSSION THREAD

**Discussion by AndreasWeste** [xanderfrangos/twinkle-tray#284] [Ideas]
The tool works great, but I live in a country where in winter the sun sets at 4 pm, and in summer at 10 pm, so i wish to change the brightness settings automatically based on my actual daylight at my longitude/latitude position.

Windows already has this implemented for its red color filter, software like f.lux can do it, and browser extension like Dark Reader also implemented this. It would be great, thank you.

**Comment by WiBla** [comment]
In addition, I'd like to know if you've ever thought about connecting the app to a photoresistor?
That way, we could replicate the "auto brightness" feature present in most smartphones.

If anyone ever finds a way to implement this, I ask that you kindly let me know, as this is something I've personally been looking out to do for quite a while now. Thanks.

**Comment by AndreasWeste** [comment]
[https://github.com/mourner/suncalc](https://github.com/mourner/suncalc)
